models/ncnews.models.js

```diff
@@ -16,3 +16,29 @@ exports.selectSelectArticleById = (article_id) => {
       return rows[0];
     });
 };
+
+exports.selectArticles = () => {
+  return Promise.all([
+    db.query("SELECT * FROM articles ORDER BY created_at DESC;"),
+    db.query("SELECT * FROM comments;"),
+  ]).then((arrayOfTables) => {
+    if (arrayOfTables[0].length === 0) {
+      return Promise.reject({ status: 404, message: "Not found" });
```

Comment:
You won't need this logic if we agree that the absence of articles would mean responding with an empty array